spec/models/post_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe Post do
+  it 'post valid' do
```

Comment:
Name test (it block) like action
```suggestion
  it 'builds post' do
```

Reply:
Also you can wrap this `it` block in context, or describe block with "#valid?" name, means you test valid method